tests/dummy/app/controllers/components-examples/flexberry-objectlistview/settings-example.js

```diff
@@ -416,4 +416,14 @@ export default ListFormController.extend({
       }
     }
   },
+
+  /**
+    Refresh page by change parameters `showEditMenuItemInRow` or `showDeleteMenuItemInRow`.
+
+    @method _pageRefresh
+    @private
+  */
+  _pageRefresh: Ember.observer('showEditMenuItemInRow', 'showDeleteMenuItemInRow', function() {
```

Comment:
Well, this is definitely not the problem solution.
Problem needs some investigation & changes in object-list-view, flexberry-menu component isn't guilty here.